app/components/app_imports_table_component.html.erb

```diff
@@ -31,6 +31,13 @@
           <% row.with_cell do %>
             <span class="nhsuk-table-responsive__heading">Import type</span>
             <%= record_type(import) %>
+            <% if import.is_a?(ClassImport) %>
+              <br />
+              <span class="nhsuk-u-secondary-text-color nhsuk-u-font-size-16">
+                <%= govuk_link_to import.session.location.name,
```

Comment:
Looks good. Only question I have is does the school session need to be linked to from the table view?

Reply:
Yeah, fair.

I think I originally added it because:

* I remember from observing the first couple of days of usage, staff were often uploading the class lists and trying to get back to the school page, often going via sessions > un/scheduled > and trying to find the school in a school list, which looked a bit frustrating.
* I think there's benefit in every mention of a school linking to the school page, similar to how (almost) all mentions of a patient are links to either the patients's global page or session page.

Having said that, I'm not convinced that the first two days are necessarily representative of regular usage, and once staff discover the link to the school from the individual upload page, they'll (hopefully) begin using that...

I've backed the link out for now.